packages/vouching/contracts/ZepToken.sol

```diff
@@ -1,28 +1,35 @@
 pragma solidity ^0.4.24;
 
-import 'openzeppelin-solidity/contracts/token/ERC20/PausableToken.sol';
-import 'openzeppelin-solidity/contracts/token/ERC20/StandardToken.sol';
+import "openzeppelin-zos/contracts/token/ERC20/ERC20Detailed.sol";
+import "openzeppelin-zos/contracts/token/ERC20/ERC20Pausable.sol";
+import "TPL-1.0-audit/contracts/TPLToken.sol";
+
+// Temporary imports for Truffle testing until TPL is released
+import "TPL-1.0-audit/contracts/ZEPValidator.sol";
+import "TPL-1.0-audit/contracts/BasicJurisdiction.sol";
+
 
 /**
  * @title ZepToken
  * @dev ZEP token contract including mintable, pausable and burnable functionalities
  */
-contract ZepToken is StandardToken, PausableToken {
+contract ZepToken is Initializable, TPLToken, ERC20Detailed, ERC20Pausable {
 
-  string public constant name = "Zep Token";
-  string public constant symbol = "ZEP";
-  uint8 public constant decimals = 18;
+  function initialize(
+    AttributeRegistry _jurisdictionAddress,
+    uint256 _validRecipientAttributeId
+  ) 
+    initializer
+    public
+  {
+    uint8 decimals = 18;
+    uint256 totalSupply = 1e8 * (10 ** uint256(decimals));
 
-  uint256 public constant TOTAL_SUPPLY = 1e8 * (10 ** uint256(decimals));
-
-  /**
-   * @dev Constructor that gives msg.sender all of existing tokens, and sets it as the owner
-   */
-  constructor() public {
-    totalSupply_ = TOTAL_SUPPLY;
-    balances[msg.sender] = TOTAL_SUPPLY;
-    emit Transfer(address(0), msg.sender, TOTAL_SUPPLY);
+    ERC20Pausable.initialize();
+    ERC20Detailed.initialize("Zep Token", "ZEP", decimals);
```

Comment:
It's a minor, but can we validate with @martriay what should be the official name and spelling for the token?

Reply:
I don't think this is highly critical considering that we can change it in the future. I'd go with "ZeppelinOS Token" + "ZEP"

Reply:
checked with him, agreed on "ZeppelinOS Token".